convert.go

```diff
@@ -195,6 +195,13 @@ func convertEvidence(c *Component, specVersion SpecVersion) {
 	}
 
 	if specVersion < SpecVersion1_6 {
+		// Spec version 1.5 uses only one Identity.
+		// cf. https://cyclonedx.org/docs/1.5/json/#components_items_evidence_identity
+		if c.Evidence.Identity != nil {
+			ids := *c.Evidence.Identity
+			ids = ids[:1]
+			c.Evidence.Identity = &ids
+		}
```

Comment:
I'm not sure about this way...
Maybe we need to select one `Identifier` using `Field` with some order

Please tell me if you have a better solution.

Reply:
IMO that is fine for now, since it keeps the insertion order.